client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -629,6 +664,72 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
+                                const desk = investigation.desk;
+                                const covidPatient = patient.covidPatientByCovidPatient;
+                                const patientCity = (covidPatient && covidPatient.addressByAddress) ? covidPatient.addressByAddress.cityByCity : '';
+                                const user = investigation.userByCreator;
+                                const county = user ? user.countyByInvestigationGroup : '';
+                                const statusReason = user ? investigation.statusReason : '';
+                                const wasInvestigationTransferred = investigation.wasInvestigationTransferred;
+                                const transferReason = user ? investigation.transferReason : '';
+                                const groupId = user ? investigation.groupId : '';
+                                const canFetchGroup = false;
+                                const subStatus = investigation.investigationSubStatusByInvestigationSubStatus ?
+                                    investigation.investigationSubStatusByInvestigationSubStatus.displayName :
+                                    '';
+                                return createRowData(
+                                    investigation.epidemiologyNumber,
+                                    investigation.coronaTestDate,
+                                    investigation.isComplex,
+                                    investigation.priority,
+                                    investigation.investigationStatusByInvestigationStatus.displayName,
+                                    subStatus,
+                                    covidPatient.fullName,
+                                    covidPatient.primaryPhone,
+                                    covidPatient.age,
+                                    patientCity ? patientCity.displayName : '',
+                                    desk,
+                                    county,
+                                    { id: user.id, userName: user.userName },
+                                    investigation.comment,
+                                    statusReason,
+                                    wasInvestigationTransferred,
+                                    transferReason,
+                                    groupId,
+                                    canFetchGroup
+                                )
+                            });
+                setAllGroupedInvestigations(allGroupedInvestigations.set(groupId, investigationRows))
+            } else {
+                investigationsByGroupIdLogger.error('Got 200 status code but results structure isnt as expected', Severity.HIGH);
+            }
+        })
+            .catch((err) => {
+                alertError('לא הצלחנו לשלוף את כל החקירות בקבוצה');
+                investigationsByGroupIdLogger.error(err, Severity.HIGH);
+            })
+            setIsLoading(false)
```

Comment:
you should do setIsLoading(false) inside the 'then' block as well.
add finaly and put it there instead of writing it twice in the 'then' and 'catch' blocks